From 259414d343f5dba9b34519f0e26bbfefd7b3f0ea Mon Sep 17 00:00:00 2001
From: Jon Nettleton <jon.nettleton@gmail.com>
Date: Mon, 31 Aug 2015 10:57:01 +0200
Subject: [PATCH] Fix versioning and defines for wayland-egl

This sets the version to 9.2.0 to match Mesa, and adds proper
defines so that software will build without extra tweaks.
---
 gpu-core/usr/lib/pkgconfig/wayland-egl.pc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/gpu-core/usr/lib/pkgconfig/wayland-egl.pc b/gpu-core/usr/lib/pkgconfig/wayland-egl.pc
index db7a17a..5b215a8 100644
--- a/gpu-core/usr/lib/pkgconfig/wayland-egl.pc
+++ b/gpu-core/usr/lib/pkgconfig/wayland-egl.pc
@@ -1,10 +1,10 @@
 prefix=/usr
-exec_prefix=/usr
-libdir=/usr/lib
-includedir=/usr/include
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
 
 Name: wayland-egl
 Description: Bind the driver EGL to the Wayland API
-Version: 1.0.0
+Version: 9.2.0
 Libs: -L${libdir} -lEGL
-Cflags: -I${includedir}
+Cflags: -I${includedir} -DEGL_API_FB -DWL_EGL_PLATFORM
-- 
1.8.3.1

